ee_tests/src/specs/quickstart_deployments.spec.ts

```diff
@@ -131,33 +131,29 @@ describe('Creating new quickstart in OSIO', () => {
     await spacePipelinePage.spaceHeader.deploymentsOption.clickWhenReady();
 
     await browser.sleep(5000);
+
+    support.info('Verifying deployments page');
+
     let spaceDeploymentsPage = new SpaceDeploymentsPage();
 
-    /* Verify that app deployed to stage presents: Success icon, blue circle graphic, text = "Output from run = 1 pods" */
+    let deploymentContainers = await spaceDeploymentsPage.getDeploymentCardContainers();
+    expect(deploymentContainers.length).toBe(1, 'number of deployed applications');
 
-    let textStr = await spaceDeploymentsPage.resourceCardByNameAndIndex(spaceName, STAGE).getText();
-    support.info('1 Output from run = ' + textStr);
-    expect(await spaceDeploymentsPage.allResourceCards.count()).toBe(8);
-    expect(await spaceDeploymentsPage.successfulDeployStatusByNameAndIndex(spaceName, STAGE).isPresent()).toBeTruthy();
+    let deploymentContainer = deploymentContainers[0];
+    expect(deploymentContainer.getApplicationName()).toBe(spaceName, 'application name');
```

Comment:
following previous comment this would become something like `expect(application.getName()).toBe(spaceName, 'application name');`